restcord/invite.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from .user import User
+
+__all__ = (
+    'Invite'
+)
+
+class Invite:
+
+    """
```

Comment:
Codacy found an issue: [One-line docstring should fit on one line with quotes (found 3) (D200)](https://app.codacy.com/manual/Yandawl/restcord.py/pullRequest?prid=5910681)